listenbrainz_spark/recommendations/create_dataframes.py

```diff
@@ -26,19 +27,41 @@ def save_dataframe_html(users_df_time, recordings_df_time, playcounts_df_time, t
             playcounts_df_time (str): TIme taken to prepare and save playcounts dataframe.
             total_time (str): Time taken to execute the script.
     """
-    lb_dump_time_window = ('{}-{}'.format(config.STARTING_YEAR, '{:02d}'.format(config.STARTING_MONTH)),'{}-{}' \
-        .format(config.ENDING_YEAR, '{:02d}'.format(config.ENDING_MONTH)))
     date = datetime.utcnow().strftime('%Y-%m-%d')
     queries_html = 'Queries-{}-{}.html'.format(uuid.uuid4(), date)
     context = {
         'users_df_time' : users_df_time,
         'recordings_df_time' : recordings_df_time,
         'playcounts_df_time' : playcounts_df_time,
-        'lb_dump_time_window' : lb_dump_time_window,
         'total_time' : total_time
     }
     save_html(queries_html, context, 'queries.html')
 
+def training_data_window():
```

Comment:
I find this function very confusing. It seems that some functionality (merging dfs) is duplicated here and in utils.get_listens(). Also, the date math in this function seems complicated. I think it might be easier to extend the functionality of get_listens to work across years and this function should really only do date calculations.